manim/utils/space_ops.py

```diff
@@ -561,16 +507,19 @@ def midpoint(
     return center_of_mass([point1, point2])
 
 
-def line_intersection(line1: Sequence[float], line2: Sequence[float]) -> np.ndarray:
-    """Returns intersection point of two lines, each defined with
+def line_intersection(
+    line1: Sequence[np.ndarray], line2: Sequence[np.ndarray]
+) -> np.ndarray:
+    """Returns intersection point of two line segments, each defined with
     a pair of vectors determining the end points.
```

Comment:
An improved version of @Kiran-Raj-Dev's suggestions.

```suggestion
    """Returns the intersection point of two lines, each defined by
    a pair of distinct points lying on the line.
```